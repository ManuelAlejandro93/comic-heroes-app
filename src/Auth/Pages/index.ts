export * from './Login';
export * from './Logout';
export * from './OnTrueLogin';
