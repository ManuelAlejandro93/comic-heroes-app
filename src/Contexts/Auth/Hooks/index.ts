export * from './useAuthHook';
export * from './onInitFunc';
