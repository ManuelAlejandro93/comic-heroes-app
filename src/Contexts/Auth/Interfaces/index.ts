export * from './AuthStateType';
export * from './ContextType';
