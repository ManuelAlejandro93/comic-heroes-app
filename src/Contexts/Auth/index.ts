export * from './AuthContext';
export * from './Reducer';
export * from './Hooks';
export * from './Interfaces';
