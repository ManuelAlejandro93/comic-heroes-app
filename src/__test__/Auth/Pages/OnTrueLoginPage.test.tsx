import { OnTrueLoggedPage } from '@/Auth';
import { render } from '@testing-library/react';

describe('pruebas en "OnTrueLoggedPage"', () => {
  test('Debe ser renderizable', () => {
    const { container } = render(<OnTrueLoggedPage />);
    expect(container).toBeTruthy();
  });
  test('Debe coincidir con el snapshot', () => {
    const { container } = render(<OnTrueLoggedPage />);
    expect(container).toMatchSnapshot();
  });
  test('Debe tener un h1 blanco que diga Welcome to Login page', () => {
    const { container } = render(<OnTrueLoggedPage />);

    expect(container.querySelector('h1')?.innerHTML).toBe(
      'Welcome to Login page'
    );
    expect(
      container.querySelector('h1')?.className.includes('text-white')
    ).toBe(true);
  });
});
