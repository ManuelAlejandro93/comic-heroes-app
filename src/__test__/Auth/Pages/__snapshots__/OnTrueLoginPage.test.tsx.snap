// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`pruebas en "OnTrueLoggedPage" Debe coincidir con el snapshot 1`] = `
<div>
  <div
    class="text-center grid pr-16 space-y-8"
  >
    <h1
      class="text-6xl text-white"
    >
      Welcome to Login page
    </h1>
    <h2
      class="text-blue-500 text-3xl"
    >
      You are already logged, enjoy.
    </h2>
  </div>
</div>
`;
